src/main/java/org/fogbowcloud/arrebol/execution/docker/DockerTaskExecutor.java

```diff
@@ -4,157 +4,223 @@
 import org.apache.log4j.Logger;
 import org.fogbowcloud.arrebol.execution.TaskExecutionResult;
 import org.fogbowcloud.arrebol.execution.TaskExecutor;
-import org.fogbowcloud.arrebol.execution.docker.exceptions.DockerCreateContainerException;
 import org.fogbowcloud.arrebol.execution.docker.exceptions.DockerRemoveContainerException;
-import org.fogbowcloud.arrebol.execution.docker.exceptions.DockerStartException;
-import org.fogbowcloud.arrebol.execution.docker.exceptions.NotFoundDockerImage;
 import org.fogbowcloud.arrebol.execution.docker.request.WorkerDockerRequestHelper;
 import org.fogbowcloud.arrebol.models.command.Command;
 import org.fogbowcloud.arrebol.models.command.CommandState;
 import org.fogbowcloud.arrebol.models.task.Task;
-import org.fogbowcloud.arrebol.models.task.TaskState;
 
 import java.io.UnsupportedEncodingException;
 import java.util.List;
 
 import static java.lang.Thread.sleep;
 
+/**
+ * This class consists of an entity of remote execution of tasks using docker containers. At each
+ * execution a new container is started and after the execution of the task is finished the
+ * container is removed. The execution of each command is not done directly by this class but by a
+ * bash script sent into the container. In cases where container initialization error occurs, the
+ * commands go to failure state and a failure result is returned.
+ */
 public class DockerTaskExecutor implements TaskExecutor {
 
-    private static final int SUCCESS_EXIT_CODE = 0;
-    private static final int FAIL_EXIT_CODE = 127;
+    private static final long poolingPeriodTime = 2000;
```

Comment:
Replace the word 'time' for the corresponding unit of measure.

Reply:
Done.